packages/runtime/src/index.ts

```diff
@@ -10,7 +10,8 @@ import { assert } from './utils/logger';
 
 export { FederationHost } from './core';
 export { registerGlobalPlugins } from './global';
-export { loadScript } from './utils';
+export { loadScript } from '@module-federation/sdk';
```

Comment:
loadScript not in utils, its exported by sdk but not exported here in utils anywhere.